docs/integration-association.md

```diff
@@ -0,0 +1,37 @@
+# Association Integration
+## Desciption
+This file try to define how the associations will be handled in our application.
+Firstly, let see some user stories linked to associations. Then, how we can implement those functionalities :
+## User Stories
+- As a user interested in philosophy, I want to be able to find related associations so that I don't have to make too much effort.
+- As a user potentially interested in joining JDRPoly, I want to be able to get enough information on this association so that I know if it interests me or not.
+- As a curious user, I want to be able to discover new associations that I never thought of so that I can start a new activity.
+- As a user, I want to be able to follow an association so that I can easily see news and events related to it.
+- As a user in charge of communication in my association, I want to be able to share latest news so that people can see what's happening in my association.
+- As a user in charge of communication in my association, I want to be able to create events and certified them with my association so that other users can see that my event is from my association.
+## Implementation
+### Dataclass Association
+We already have a dataclass called Association. Currently, it has the following attributes : an "associationId", a "name" and a "description". We can add to this :
+- "relatedTags: Set<Tag>", the set of all tags related to the association
+- "moderators: Set<User>", the set of people that have the right to publish actuality, modify association description, add members and create events in the name of the association
```

Comment:
there is actually a relation on the database (already present), specifying the committeeMembers of an association (n-to-n relation between `UserProfile`s and `Association`s). For now the property is already available on the `UserProfile` as the attribute `committeeMember`. This is needed for the Event screen to know for which Association the currently logged in User has the rights to create events for (also validated by the RLS Policy for inserts and updates on Events). I don't think having it on the Association makes sense since it would mean we'd need to fetch and add it to every Event displayed (since Events include the Association object in the `organizer` field. So far I can't see any need to have that information available since we don't plan to display it.

Reply:
I was not sure on how the comiteeMember was handled, thanks for the explanation. It makes sense to remove this attribute from Association object, I will correct it